examples/process-compose/devenv.nix

```diff
@@ -0,0 +1,30 @@
+{ pkgs, ... }:
+
+{
+  process.implementation = "process-compose";
+
+  processes.foo.exec = "echo foo; sleep 5";
+
+  # implicitly creates a postgres process
+  # consider https://github.com/cachix/devenv/issues/75
```

Comment:
I left a comment on that issue, let's leave this discussion there and not pollute the example :)